recipe/meta.yaml

```diff
@@ -127,6 +127,11 @@ about:
   description: |
     MPICH is a high performance and widely portable implementation of the
     Message Passing Interface (MPI) standard.
+
```

Comment:
I'm not sure this is the right place to document the use of UCX vs OFI.
Maybe this should be added to a post-activate script that prints the information upon environment activation, like we do in Open MPI.

Reply:
The thing is that when you install a lot of conda packages, the messages does not stay long on the screen and is very difficult to spot, whereas if it was added in the description section it would show up at the bottom of the https://anaconda.org/conda-forge/mpich page.